packages/mco/hooks/subscription.ts

```diff
@@ -25,43 +34,100 @@ import {
   DisasterRecoveryResourceKind,
 } from './disaster-recovery';
 import {
+  getApplicationResourceObj,
   getPlacementDecisionsResourceObj,
   getPlacementResourceObj,
   getPlacementRuleResourceObj,
   getSubscriptionResourceObj,
 } from './mco-resources';
 
-const getResources = (namespace: string) => ({
-  placements: getPlacementResourceObj({ namespace }),
-  placementDecisions: getPlacementDecisionsResourceObj({ namespace }),
-  placementRules: getPlacementRuleResourceObj({ namespace }),
-  subscriptions: getSubscriptionResourceObj({ namespace }),
+const getResources = () => ({
+  applications: getApplicationResourceObj(),
+  placements: getPlacementResourceObj(),
+  placementDecisions: getPlacementDecisionsResourceObj(),
+  placementRules: getPlacementRuleResourceObj(),
+  subscriptions: getSubscriptionResourceObj(),
 });
 
-const getPlacementMap = (placements: ACMPlacementType[]) => {
-  return (
-    placements.reduce(
-      (arr, placement) => ({
-        ...arr,
+const appFilter = (application: ApplicationKind) =>
+  application?.spec?.componentKinds?.some(
+    (componentKind) =>
+      componentKind?.group === ACMSubscriptionModel?.apiGroup &&
+      componentKind?.kind === ACMSubscriptionModel?.kind
```

Comment:
don't use this just everywhere, use where it is needed...
```suggestion
      componentKind.group === ACMSubscriptionModel.apiGroup &&
      componentKind.kind === ACMSubscriptionModel.kind
```

Reply:
done